documentation/source/quickstart.rst

```diff
@@ -10,7 +10,7 @@ Pre-requisites
 
 Cocotb has the following requirements:
 
-* Python 2.7, Python 3.5+ (recommended)
+* Python 3.5+
```

Comment:
Add a comment here like "(Use cocotb < 1.4 if you need Python 2.7 support.)

Reply:
Is there a permalink to the old docs we could point to as well?

Reply:
yes: https://cocotb.readthedocs.io/en/v1.3.0/

Reply:
It's in the bottom left corner.